packages/date-picker/src/vaadin-date-picker-overlay-content.js

```diff
@@ -767,7 +773,10 @@ class DatePickerOverlayContent extends ControllerMixin(ThemableMixin(DirMixin(Po
 
   __onTodayButtonKeyDown(event) {
     if (this.hasAttribute('fullscreen')) {
-      event.stopPropagation();
+      // Do not prevent closing on Esc
+      if (!event.key === 'Escape') {
```

Comment:
Looks suspicious. Should it be `event.key !=== 'Escape'` or `event.key === 'Escape'`?

Reply:
My bad, fixed 🤦‍♂️